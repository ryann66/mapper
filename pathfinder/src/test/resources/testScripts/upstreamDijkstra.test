# Test finding path against node direction

CreateGraph mg

AddNode mg A
AddNode mg B
AddEdge mg B A 1.0

FindPath mg A B

AddNode mg C
AddNode mg D
AddEdge mg A D 5.0
AddEdge mg D C 5.0
AddEdge mg C B 5.0

FindPath mg A B