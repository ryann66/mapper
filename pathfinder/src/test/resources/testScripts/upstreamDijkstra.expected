# Test finding path against node direction

created multigraph mg

added node A to mg
added node B to mg
added edge 1.000 from B to A in mg

path from A to B:
no path found

added node C to mg
added node D to mg
added edge 5.000 from A to D in mg
added edge 5.000 from D to C in mg
added edge 5.000 from C to B in mg

path from A to B:
A to D with weight 5.000
D to C with weight 5.000
C to B with weight 5.000
total cost: 15.000